client/src/components/RatingsNReviews/ReviewList.jsx

```diff
@@ -1,9 +1,62 @@
 import React from "react";
-
+// import axios from 'axios';
+import Sample from '/home/yanlin/hackreactor/Project-Atelier/example/reviews.js'
```

Comment:
This path doesn't seem like a secure thing to write. Will you be replacing it later?

Reply:
I will change it later